addons/google_calendar/models/calendar_attendee.py

```diff
@@ -39,5 +39,13 @@ def do_decline(self):
         return res
 
     def _sync_event(self):
+        # For weird reasons, we can't sync status when we are not the responsible
+        # We can't adapt google_value to only keep ['id', 'summary', 'attendees', 'start', 'end', 'reminders']
+        # and send that. We get a Forbidden for non-organizer error even if we only send start, end that are mandatory !
+        all_events = self.mapped('event_id').filtered(lambda e: e.google_id)
+        other_events = all_events.filtered(lambda e: e.user_id and e.user_id.id != self.env.user.id)
+        for user in other_events.mapped('user_id'):
+            service = GoogleCalendarService(self.env['google.service'].with_user(user))
+            other_events.filtered(lambda ev: ev.user_id.id == user.id).with_user(user)._sync_odoo2google(service)
```

Comment:
I was curious so I ran the profiler on both versions. I commented the call to _sync_odoo2google(service) to avoid measuring the request time.  I saved the results in the following archive.
I tested it on 720 events (recurrence 'forever') that enters the `other_events` recordset. 
`(all_events - other_events)` was empty. 

[speedoscope.zip](https://github.com/odoo/odoo/files/7008053/speedoscope.zip)

It seems that your version is ~4 time slower.

Reply:
Oh, sorry, thanks for the information